lib/addons/src/types.ts

```diff
@@ -7,6 +7,8 @@ export enum types {
   TOOL = 'tool',
   PREVIEW = 'preview',
   NOTES_ELEMENT = 'notes-element',
+  IFRAME_START = 'iframe_start',
```

Comment:
Sorry, I don't think this API is a good idea.

I want to be super careful with adding addon types, as we'll be stuck with supporting them forever.

I'd rather have a single IframeWrapper type addon, which I might have a usecase for when i want to extract the zoom functionality into an addon.

I think you can do everything in a wrapper you could do with a pre & post